nbconvert/preprocessors/execute.py

```diff
@@ -38,12 +38,14 @@ class ExecutePreprocessor(Preprocessor):
     Executes all the cells in a notebook
     """
 
-    timeout = Integer(30, config=True,
+    timeout = Integer(10, config=True, allow_none=True,
```

Comment:
No, it should be changed back to 30. I told @sandyshao to open even an unfinished PR to get used to Git/GitHub and get review / interact with you guys a bit.